test/tests/haskell-ghci/container.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+echo "" | ghci
```

Comment:
Ohhh ok.  You don't want to adapt it to make sure "ghci" does something
meaningful instead? :)